recipes-tests/pom.xml

```diff
@@ -177,6 +182,11 @@
             <scope>test</scope>
             <type>test-jar</type>
         </dependency>
+
+        <dependency>
+            <groupId>io.smallrye.config</groupId>
+            <artifactId>smallrye-config</artifactId>
```

Comment:
Shouldn't it be declared with a `test` scope?

Reply:
Yes it should.

I pushed a commit, let's wait for CI now.